frontend/drink_list_picker.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+
+
+async function login_user() {
```

Comment:
This doesn't look like the right name for this function

Reply:
fixed

Reply:
It's better, but I think we need a new convention for naming functions